proposal.tex

```diff
@@ -1,2 +1,30 @@
 \section{Community Proposal Process}\label{sec:process}
-% Melissa
\ No newline at end of file
+% Melissa
+
+\textbf{Timeline --}
+As the distribution of subthreshold alerts is planned to begin in year 2 of the LSST, the first community proposals will be solicited during the first half of year 1.
+The first deadline will be timed to match the midpoint of year 1 (i.e., the observational cutoff date for Data Release 1).
+Thereafter, proposals would be solicited on a yearly basis.
+
+\textbf{Proposals --}
+The community will be asked to provide proposals for subthreshold alerts that includes three components:
+
+\begin{enumerate}
+\item The anticipated science impact, with a motivation regarding why the science goals require alerts within 60 seconds and cannot be achieved using the Prompt Products Database, which is updated within 24 hours.
+\item A detailed description of how the subthreshold alerts would be identified, e.g., lists of coordinates with inclusion radii, sky bounding box, or other difference-image source detection or measurement parameters.
+\item A consideration of the technical considerations that speaks to the feasibility of the proposed subthreshold alerts.
+\end{enumerate}
+
+In keeping with best practices for research inclusion, the proposals will be anonymized (e.g., using the NOIRLab TAC system or similar).
+
+As alerts are the only fully public LSST data product, the proposals will also be public.
+
+A more detailed call for proposals and a template will be provided at the start of LSST Operations.
+
+\textbf{Evaluation --} A technical review for proposal feasibility will be done first, by a small committee within Rubin Data Production team that is headed by the lead of Alert Production.
```

Comment:
> within Rubin Data Production team
Although it will probably mostly be RDP who will carry out the  feasibility study, I wouldn't like to rule out bringing in expertise from RPF/ROO/EPO at this stage, so could we just say "a small committee within Rubin Operations"

Reply:
(also note Data Production is a Department, not a Team")

Reply:
made the change to just say "within Rubin Operations".